teuthology/task/selinux.py

```diff
@@ -53,6 +53,9 @@ def filter_hosts(self):
             if remote.is_vm:
                 msg = "Excluding {host}: VMs are not yet supported"
                 log.info(msg.format(host=remote.shortname))
+            elif remote.os.name in ['opensuse', 'sle']:
+                msg = "Excluding {host}: opensuse, sle are not yet supported"
```

Comment:
maybe change to?
"Excluding {host}: SELinux is not supported for '{os}' os_type yet"

Reply:
done